fontforge/svg.c

```diff
@@ -2777,7 +2777,36 @@ return( NULL );
 return( eret );
 }
 
-static Entity *SVGParseSVG(xmlNodePtr svg,int em_size,int ascent,bool scale) {
+void SCDimensionFromSVG(xmlNodePtr svg, SplineChar *sc, bool vert) {
+    char *num;
+    double width, height;
+    num = (char *) xmlGetProp(svg,(xmlChar *) "width");
+    if ( num!=NULL ) {
+	width = strtod(num,NULL);
```

Comment:
I'm not necessarily against this PR as-is because particular workflows will use particular conventions and we can put the onus on the user. However, we should probably discuss this a bit.

As I understand it any of these are valid SVG width/height entries:

- 640
- 80%
- 5cm

Taking the value from one of the latter cases seems like a mistake. 

Taking @ctrlcctrlv's note into account as well, one could imagine using something like the following algorithm:

```
If "width" attribute is present
    width = strtod(num, &endptr);
    if endptr points to the end of the string (or whitespace only?)
       found = true
if not found
    check viewBox for width
    if it has one, grab it and found = True
if not found:
    Emit "dimension" flag-related warning
```

Reply:
Agreed. Will patch.